content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+@acerbi_role_2016 further modelled such error-correcting redundancy, simulating minimal scenarios that could favour its evolution given fixed cost-reward constraints.
+@eriksson_corpses_2014 also decomposed real world transmission into three phases:
+choose-to-receive, encode-and-retrieve, and choose-to-transmit.
+Focusing on emotional selection studies showing that participants are more willing to pass on stimuli that elicit disgust [@heath_emotional_2001], the authors showed that any of those phases can be the target of a selection pressure.
+Such pressures can additionally counteract each other, showing that the transmission process is more complex than was initially assumed.
+Studies of digital media have adopted the increased level of detail, as digital communication lets users copy-paste and bypass the encode-and-retrieve phase of transmission [@acerbi_cultural_2016].
+
+Methodological exploration has also led to more free-form interaction setups:
+in a study of the transmission of risk perception, @moussaid_amplification_2015 chose to make the interaction and underlying transmission of information an open-ended process.
+After initiating a chain by providing the first participant with a set of documents to read on their own, later participants were left to talk freely in successive dyads, and the whole session was recorded on film for subsequent analysis.
+An analogous change of setup was operated by @caldwell_studying_2008 [-@caldwell_experimental_2008] who investigated the cumulative aspect of the evolution of building techniques in a series of experiments asking participants to construct spaghetti towers or paper planes, later evaluated by their height and flight distance.
+A crucial point in those setups, Sterelny notes [-@sterelny_cultural_2017, n. 12], is that participants could observe the preceding generation during their experimenting and building, meaning they had access to rich context for the learning phase of transmission.
+Taking another step to embrace interaction with a setup where participants had to repeatedly build Lego cars in pairs (though without transmission), @mcgraw_cultures_2014 take the complete joint interaction to be their object of study and propose to use the constructions resulting from interacting dyads as a trace of the processes that took place in the interaction.
+By looking at the characteristics of the cars built by participants, the authors claim to present "methods for discerning, and quantifying, schema-like intersubjective understandings in material form" [@mcgraw_cultures_2014, p. 4; see also @mitkidis_building_2015; and @wallot_beyond_2016], effectively integrating critiques that I discuss below in @sec:intro-review-criticisms.
```

Comment:
### Camille
Remarque générale, éviter de renvoyer aux sections suivantes, "integrating critiques that I discuss below in Section 1.1.5."; idéalement il ne faudrait le faire qu'en cas d'absolue nécessité, donc plutôt expliquer immédiatement le point ou bien le discuter plutôt effectivement dans la section ultérieure sans faire de happening.